frontend/src/components/atoms/inputs/DateInput.js

```diff
@@ -1,9 +1,57 @@
-import React from 'react';
-import Input from './Input';
+import { useState } from 'react';
+import Label from './Label';
+import Datetime from 'react-datetime';
+import moment from 'moment';
 
-export const DateInput = (props) => (
-  // the placeholder is used only on Safari and IE11 where input[type="date"] is not implemented.
-  <Input {...props} placeholder={'AAAA-MM-JJ'} type="date" />
-);
+import './DateInput.css';
+import 'react-datetime/css/react-datetime.css';
+
+export const DateInput = ({
+  label = '',
+  helper = '',
+  name = '',
+  placeholder = 'Sélectionnez une date',
+  value = '',
+  disabled = false,
+  onChange,
+  required = false,
+  inputStyle = null,
+  ...rest
+}) => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <Datetime
+      open={open}
+      value={value}
+      onChange={(date) => {
+        const formattedDate = moment(date).format('DD/MM/YYYY');
+        onChange({
+          target: {
+            name,
+            value: formattedDate,
+          },
+        });
+        setOpen(false);
+      }}
+      renderInput={() => {
+        return (
+          <div className="fr-input-group" style={inputStyle}>
```

Comment:
Use a react fragment here and use the group as a wrapper for the entire component.

```suggestion
          <>
```